users/sigma/sigma.c

```diff
@@ -0,0 +1,276 @@
+/*
+  Copyright 2018 Yann Hodique <yann.hodique@gmail.com> @sigma
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 2 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "sigma.h"
+
+userspace_config_t runtime_userspace_config;
```

Comment:
Just a heads up, but the EEPROM stuff has been merged into the main QMK code, so you can use that, if you like: 
https://docs.qmk.fm/#/custom_quantum_functions?id=persistent-configuration-eeprom